test/napi/napi-app/standalone_tests.cpp

```
@@ -2794,6 +2794,74 @@ static napi_value test_external_buffer_with_pending_exception(
+    external_for_transfer_finalized_off_thread++;
+  }
+  external_for_transfer_finalized++;
+  delete owner;
+  free(data);
+}
+
+// The bytes are 1, 2, 3, ... so a copy made on another thread can be checked.
+static uint8_t *external_for_transfer_bytes(size_t length) {
+  auto *bytes = static_cast<uint8_t *>(malloc(length));
+  for (size_t i = 0; i < length; i++) {
+    bytes[i] = static_cast<uint8_t>(i + 1);
```

Cppcheck (2.21.0)

[warning] If memory allocation fails, then there is a possible null pointer dereference

(nullPointerOutOfMemory)